### PR TITLE
5301 - Add email template for refunds

### PR DESCRIPTION
**Pull request by sutherlanda**
*Issue #:* 5301
https://github.com/bcgov/entity/issues/<Put the github issue number here>

*Description of changes:*

Adds an email template for refund requests.

Note: unable to test locally as I didn't have the documentation for setting up an environment.  Let me know if anything is off and I can address in this PR.

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by sumesh-aot** [comment]
Looks good. There are some lint errors, also please add some test cases.

**Review by sumesh-aot — APPROVED** [review]

**Comment by sutherlanda** [comment]
> Looks good. There are some lint errors, also please add some test cases.

Thanks @sumesh-aot .  I have pytests, but I am not able to verify that they work because pytest will not run without an environment file.  Can either you or @saravanpa-aot please provide me with a sample .env so I know how to set that up locally?  Thanks.

**Comment by saravanpa-aot** [comment]
> > Looks good. There are some lint errors, also please add some test cases.
> 
> Thanks @sumesh-aot . I have pytests, but I am not able to verify that they work because pytest will not run without an environment file. Can either you or @saravanpa-aot please provide me with a sample .env so I know how to set that up locally? Thanks.

sent you in rocket chat

File: queue_services/account-mailer/src/account_mailer/email_processors/refund_requested.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    template = Path(f'{current_app.config.get("TEMPLATE_PATH")}/REFUND_REQUEST.html').read_text()
+    filled_template = substitute_template_parts(template)
+
+    # render template with vars from email msg
+    jnja_template = Template(filled_template, autoescape=True)
+    html_out = jnja_template.render(
+        request=email_msg
+    )
+    return {
+        'recipients': current_app.config.get("REFUND_REQUEST").recipients,
```
Comment (saravanpa-aot) [inline queue_services/account-mailer/src/account_mailer/email_processors/refund_requested.py:38]: Its an array of items or just csv?  current_app.config.get("REFUND_REQUEST").recipients  is correct or just current_app.config.get("REFUND_REQUEST") enough?

**Review by saravanpa-aot — APPROVED** [review]
left a minor comment

File: queue_services/account-mailer/src/account_mailer/worker.py [hunk 2]
```diff
@@ -45,36 +46,42 @@
...
+            email_dict = payment_completed.process(email_msg)
+            
+        elif message_type == 'bc.registry.payment.refundRequest':
             email_msg = json.loads(event_message.get('data'))
-            logger.debug('Extracted email msg: %s', email_msg)
-            process_email(email_msg, FLASK_APP)
+            email_dict = refund_requested.process(email_msg)
 
+        logger.debug('Extracted email msg: %s', email_dict)
+        process_email(email_msg, FLASK_APP)
```
Comment (nitheesh-aot) [inline queue_services/account-mailer/src/account_mailer/worker.py:68]: this should be 'email_dict' instead of email_msg right?

Comment (sutherlanda) [reply queue_services/account-mailer/src/account_mailer/worker.py]: Yes, thank you.  Fixed.

File: queue_services/account-mailer/src/account_mailer/email_processors/refund_requested.py (hunk 1 above)
Comment (sutherlanda) [reply queue_services/account-mailer/src/account_mailer/email_processors/refund_requested.py]: It would just be a string of emails seperated with ';'

**Pull request merged**